ai/ai21labs/v0/config/tasks.json

```diff
@@ -0,0 +1,1056 @@
+          "description": "Type of each segment of the input text",
+          "instillUIOrder": 0,
+          "title": "Segment Type"
+        }
+      },
+      "required": [
+        "summerizations"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_TEXT_PARAPHRASING": {
+    "instillShortDescription": "Paraphrase text content",
+    "description": "The model returns 10 possibilities for the requested text. AI21 Studio's Paraphrase API offers access to our state-of-the-art paraphrasing engine. It has been fine-tuned specifically for paraphrasing. As a result, it's easier to integrate into your systems, and since it's optimized for this purpose, it's also more efficient (and therefore cheaper) than building it from scratch. This is also the engine behind Wordtune - so you know it's pretty awesome.",
+    "input": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "text",
+        "style"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "text": {
+          "description": "The text content to be paraphrased",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIMultiline": true,
+          "instillUIOrder": 1,
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "title": "Text",
+          "type": "string"
+        },
+        "style": {
+          "description": "The style of the text",
+          "enum": [
+            "general",
+            "casual",
+            "formal",
+            "long",
+            "short"
+          ],
+          "example": "general",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIOrder": 2,
+          "instillUpstreamTypes": [
+            "value"
+          ],
+          "title": "Style",
+          "type": "string"
+        },
+        "start-index": {
+          "description": "The start index of the text to be paraphrased",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 3,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Start Index",
+          "type": "integer"
+        },
+        "end-index": {
+          "description": "The end index of the text to be paraphrased",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 4,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "End Index",
+          "type": "integer"
+        }
+      },
+      "required": [
+        "text",
+        "style"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Output",
+      "instillUIOrder": 0,
+      "properties": {
+        "suggestions": {
+          "description": "Model Output",
```

Comment:
nit
clearer?
`"description": "Model Output",`

`The suggestions of the text to be paraphrased from model`

Reply:
fixed